src/screens/Ticketing/Purchase/Payment/SelectPaymentMethod/index.tsx

```diff
@@ -300,6 +300,7 @@ const PaymentOptionView: React.FC<PaymentOptionsProps> = ({
       <TouchableOpacity
         style={[styles.paymentOption, styles.centerRow]}
         onPress={select}
+        accessibilityLabel={info.a11y}
         accessibilityHint={info.a11y}
```

Comment:
Man kan også legge inn `accessibilityRole` og `accessibilityState`, så er det tydelig at det er en radio button som er valgt eller ikke 😊

```suggestion
        accessibilityHint={info.a11y}
        accessibilityRole="radio"
        accessibilityState={{selected: selected}}
```